duck/duckdb.go

```diff
@@ -278,6 +278,9 @@ func exclude(query string) (bool, string) {
 			return true, exclusion
 		}
 	}
+	// check for "set" command
+	// can't just add to exclusions since that uses Contains
+	// select * from corset - should be ok
```

Comment:
You could argue that the same applies to all the other commands too, right? like `st_read` for instance.

An LLM tells me that we could develop
> Develop a custom query parser to analyze and sanitize user input before execution:
> 1. Restrict allowed SQL keywords and functions
> 2. Validate table and column names against a predefined list
> 3. Ensure JOIN conditions use only permitted tables

I guess that's a bigger lift, but it sounded more watertight to me 🤔

Reply:
yeah this is really a quick fix.  yeah `st_read` could possibly be valid.  `select * from first_reader` or something.  i will fix that